File: hw22/sqrts.py
# print square roots in Python language.  R. Brown, 9/2010 

from math import sqrt
import sys

if (len(sys.argv))>1:
  max=int(sys.argv[1])
  print("sqrt(n)\n--------")
  for n in range(0, max+1):
    print(sqrt(n))
else:
	print("Enter atleast 1 argument")

File: hw22/Sqrts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

import java.io.*;

class Sqrts {

  public static void main(String args[]) {

  	if(args.length>0){
  	System.out.println("sqrt(n)");
    System.out.println("--------");
    int n;
    int max=Integer.parseInt(args[0]);
    for (n=0;  n<=max;  n++)
      System.out.println(Math.sqrt(n));
	
}else{
	System.out.println("Please Enter atleast 1 argument");
}
        return;

  }
}